figures/scripts/TelomeraseActivitites.R

```diff
@@ -0,0 +1,174 @@
+library('ggpubr')
+stringsAsFactors=FALSE
+library(stringr)
+library(gridBase)
+library(gridGraphics)
+library(optparse)
+library(forcats) ### for fct_reorder()
+
+###################################################################################################
+
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git")) 
+
+A = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAPolya_counts.txt")   ### Variable representing Telomerase activity-prediction for Polya_counts
+				 
+B = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAStranded_counts.txt")  ### Variable representing Telomerase activity-prediction for Stranded_counts
+				 
+C = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAPolya_FPKM.txt")      ### Variable representing Telomerase activity-prediction for PolyA_FPKM data
+				 
+D = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAStranded_FPKM.txt")   ### Variable representing Telomerase activity-prediction for Stranded_FPKM data
```

Comment:
> Maybe I got a bit lost in the review of the shell script, but my understanding was that the output of the EXTEND steps would be in figures/results/. These lines indicate that you are using the results files in analyses/telomerase-activity-prediction/results to make your figures, is that correct?

Yeah that's right I am using the results from analyses/telomerase-activity-prediction/results to make figures. It means I need to make a results folder under figures directory. But in that case I need to run all the analysis section again to generate EXTEND scores first, save them in figures/results and then generate figures. But it would replicate the same thing which we did for analysis. If the point is to use updated results based on data then why not to save them under the analysis directory (I mean regenerate them over there) and then use the updated result. What do you suggest?

Reply:
To make sure we're on the same page - in the shell script addition above, you are putting new results in `figures/results` with:

```
Rscript --vanilla 01-run-EXTEND.R --input ../../data/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds --output results/TelomeraseScores_PTBAStranded_FPKM.txt
Rscript --vanilla 01-run-EXTEND.R --input ../../data/pbta-gene-expression-rsem-fpkm-collapsed.polya.rds --output results/TelomeraseScores_PTBAPolya_FPKM.txt
Rscript --vanilla 01-run-EXTEND.R --input ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed.stranded.rds --output results/TelomeraseScores_PTBAStranded_counts.txt
Rscript --vanilla 01-run-EXTEND.R --input ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed.polya.rds --output results/TelomeraseScores_PTBAPolya_counts.txt
```

Where `01-run-EXTEND.R` runs the step that calculates EXTEND scores. But you are not using them in the R script you are adding here that generates the figure. You are instead using `analyses/telomerase-activity-prediction/results`. Is that correct?

I just noticed that I think that would also need to be `${analyses_dir}/telomerase-activity-prediction/01-run-EXTEND.R` to run in the figure generation script.

Reply:
Oh I just noticed the mistake, you are right, in the shell script I did what you mentioned while the R script didn't use those results instead I directed it to the analysis directory. I will do all the changes as you mentioned and will resubmit. Thanks for the details.

Further I have a question about the supplementary figure, So the supplementary figure would be placed under the figures directory or it would be separate.

Reply:
> Further I have a question about the supplementary figure, So the supplementary figure would be placed under the figures directory or it would be separate.


We don't have a robust policy for that yet, so let's have you place the figure in `figures` but prefix the PNG file name with `supplementary_` for now.